cafebabel/articles/models.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+import os
+from hashlib import md5
+from pathlib import Path
+
+from mongoengine import signals
+
+from .. import app, db
+from ..core.helpers import slugify
+from ..users.models import User
+
+
+class Article(db.Document):
+    title = db.StringField(required=True)
+    slug = db.StringField(required=True)
+    language = db.StringField(max_length=2, required=True)
+    category = db.StringField()
+    summary = db.StringField()
+    body = db.StringField(required=True)
+    image = db.StringField()
+    status = db.StringField(default='draft')
+    uid = db.StringField(required=True)
+    editor = db.ReferenceField(User, reverse_delete_rule=db.NULLIFY)
+    author = db.ReferenceField(User, reverse_delete_rule=db.NULLIFY)
+    creation_date = db.DateTimeField(default=datetime.datetime.utcnow)
+    publication_date = db.DateTimeField()
+    _upload_image = None
+
+    def __str__(self):
+        return self.title
+
+    @property
+    def is_draft(self):
+        return self.status == 'draft'
+
+    @property
+    def is_published(self):
+        return self.status == 'published'
+
+    @property
+    def image_url(self):
+        if self.image:
+            return f'{app.config.get("ARTICLES_IMAGES_URL")}/{self.image}'
```

Comment:
You should be able to do directly `return app.config.get("ARTICLES_IMAGES_URL") / self.image`